src/data/staff.json

```diff
@@ -0,0 +1,78 @@
+{   
+   "superadmins": [ 
+        [
+            "brussels",
+            "https://cdn.discordapp.com/avatars/608641641284370462/9c6698249645077608227af19f0673b2.webp?size=80"
+        ],
```

Comment:
This works okay, but I think this would be better as an object instead of an array.  That way we can access by key when you map through them:
```
user["name"], user["imgUrl"], user["bio"]...
```